src/components/card/z-card-info/index.tsx

```diff
@@ -89,46 +127,65 @@ export class zCardInfo {
     }
   }
 
-  renderOnlineLicenzeSection() {
+  renderOnlineLicenseSection() {
     if (!this?.cardData?.onlineLicense) return;
 
     return (
-      <section>
+      <section ref={(el) => (this.onlineLicenseWrapper = el)}>
         <span class="license-heading">
           <span>Licenza online</span>
           {this.setExpirationLicenseMessage("online")}
         </span>
-        Scadenza: <b>{this.cardData.onlineLicense.expiration}</b>
+        Scadenza il <b>{this.cardData.onlineLicense.expiration}</b>
         <br />
       </section>
     );
   }
 
-  renderOfflineLicenzeSection() {
+  renderOfflineLicenseSection() {
     if (!this?.cardData?.offlineLicense) return;
 
     return (
-      <section>
+      <section ref={(el) => (this.offlineLicenseWrapper = el)}>
         <span class="license-heading">
           <span>Licenza offline</span>
           {this.setExpirationLicenseMessage("offline")}
         </span>
-        Scadenza: <b>{this.cardData.offlineLicense.expiration}</b>
+        Scadenza il <b>{this.cardData.offlineLicense.expiration}</b>
         <br />
         Installazioni disponibili:{" "}
         <b>{this.cardData.offlineLicense.installations}</b>
       </section>
     );
   }
 
+  componentDidRender() {
+    if (
```

Comment:
questo snippet si può astrarre in una fn di utility che ritorna l'height?

Reply:
Intendi una funzione all'interno del componente? Direi proprio di sì

Reply:
dicevo proprio una funzione globale in tools, ma vedo adesso che è un pò complicato astrarre tutto

Reply:
No, non c'è nulla di globale in questa funzione 😅  Comunque l'ho spostata in un metodino, così lasciamo in componentDidRender più pulito